web/themes/joinup/joinup_theme.theme

```diff
@@ -202,6 +209,104 @@ function joinup_theme_preprocess_field_multiple_value_form(&$variables) {
 }
 
 /**
+ * Implements hook_preprocess_HOOK() for form-element.html.twig.
+ */
+function joinup_theme_preprocess_form_element(&$variables) {
+  if (isset($variables['label']['#title']) && method_exists($variables['label']['#title'], 'getArguments')) {
+    $arguments = $variables['element']['#title']->getArguments();
+
+    // Array contains form elements, which are displayed with floating label.
+    $floating_label_types = [
+      'email',
+      'entity_autocomplete',
+      'password',
+      'textfield',
+      'url',
+    ];
+
+    if (in_array($variables['type'], $floating_label_types)) {
+      // Make label visible, so it can be displayed inside floating field.
+      $variables['label']['#title_display'] = 'visible';
+
+      // Pass element title to label.
+      if (!empty($arguments['@title'])) {
+        $variables['label']['#title'] = $arguments['@title'];
+      }
+    }
+  }
+}
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter() for input.html.twig.
+ */
+function joinup_theme_theme_suggestions_input_alter(array &$suggestions, array $variables) {
+  // Get input untranslated input value.
```

Comment:
This hook gets called very very often, because a lot of form elements are input. Better try to early return: since we are handling only buttons, I would use
```php
if (empty($variables['element']['#is_button'])) {
  return;
}
```